config/_default/menus/main.en.yaml

```diff
@@ -5617,11 +5617,6 @@ menu:
       weight: 3
       params:
         skip: true
-    - name: Connect Logs and Traces
```

Comment:
This entry was added via https://github.com/DataDog/documentation/commit/5b6f4e80cc1c60cba36996f285358b0402e8ca9d, which was directly pushed to the default branch. As a result, I’m unsure why it was placed under `Log Management` 🙏